atomicapp/cli/main.py

```diff
@@ -52,9 +52,10 @@ def print_app_location(app_path):
 
 def cli_genanswers(args):
     argdict = args.__dict__
+    location = argdict['answers']
```

Comment:
I was following similar to how we do:
```
destination = argdict['destination'] 
```
Should we not follow this convention / convert to do this later in main.py?